tests/python_client/testcases/test_query.py

```diff
@@ -600,8 +600,7 @@ def test_query_output_float_vec_field(self):
 
     @pytest.mark.tags(CaseLabel.L2)
     @pytest.mark.parametrize("wildcard_output_fields", [["*"], ["*", default_float_field_name],
-                                                        ["*", default_int_field_name],
-                                                        ["%"], ["%", default_float_field_name], ["*", "%"]])
```

Comment:
what is the behavior of "%", any case cover this behavior?

Reply:
> what is the behavior of "%", any case cover this behavior?

'%' is invalid now with message `field % not exist`

Reply:
I have added the check cases of invalid output fields